flow/designs/gf12/coyote/metadata-hier_rtlmp-ok.json

```diff
@@ -31,89 +31,89 @@
-  "cts__timing__setup__ws__post_repair": 105.29,
-  "cts__timing__drv__max_slew__post_repair": 10.0,
-  "cts__timing__drv__max_fanout___post_repair": 1831.0,
+  "cts__timing__setup__ws__pre_repair": -164.52,
+  "cts__timing__setup__tns__post_repair": -4956.01,
+  "cts__timing__setup__ws__post_repair": -166.96,
+  "cts__timing__drv__max_slew__post_repair": 23.0,
+  "cts__timing__drv__max_fanout___post_repair": 1839.0,
   "cts__timing__drv__max_cap__post_repair": 0.0,
   "cts__timing__setup__tns": 0.0,
-  "cts__timing__setup__ws": 56.53,
-  "cts__timing__drv__max_slew": 7.0,
-  "cts__timing__drv__max_fanout": 1831.0,
-  "cts__design__instance__count__hold_buffer": 285327.0,
-  "globalroute__clock__latency__min": 368.74,
-  "globalroute__clock__latency__max": 499.07,
-  "globalroute__clock__skew__worst": 115.87,
-  "globalroute__timing__setup__tns": 0.0,
-  "globalroute__timing__setup__ws": 13.74,
-  "globalroute__timing__drv__max_slew": 1.0,
-  "globalroute__timing__drv__max_fanout": 1831.0,
+  "cts__timing__setup__ws": 2.63,
+  "cts__timing__drv__max_slew": 26.0,
+  "cts__timing__drv__max_fanout": 1839.0,
+  "cts__design__instance__count__hold_buffer": 285701.0,
+  "globalroute__clock__latency__min": 389.2,
+  "globalroute__clock__latency__max": 510.12,
+  "globalroute__clock__skew__worst": 106.0,
+  "globalroute__timing__setup__tns": -1341.64,
+  "globalroute__timing__setup__ws": -67.49,
+  "globalroute__timing__drv__max_slew": 0.0,
+  "globalroute__timing__drv__max_fanout": 1839.0,
   "globalroute__timing__drv__max_cap": 0.0,
-  "globalroute__timing__clock__slack": 13.744,
-  "globalroute__timing__clock__period": 3786.944,
-  "globalroute__runtime__total": "3:49.49",
-  "globalroute__cpu__total": 221.55,
-  "globalroute__mem__peak": 5885548.0,
-  "detailedroute__route__wirelength": 5767138.0,
-  "detailedroute__route__vias": 5192486.0,
+  "globalroute__timing__clock__slack": -67.486,
+  "globalroute__timing__clock__period": 3864.112,
+  "globalroute__runtime__total": "6:34.01",
+  "globalroute__cpu__total": 385.93,
+  "globalroute__mem__peak": 5898432.0,
+  "detailedroute__route__wirelength": 5786389.0,
+  "detailedroute__route__vias": 5376574.0,
   "detailedroute__route__drc_errors": 0,
-  "detailedroute__runtime__total": "2:10:01",
-  "detailedroute__cpu__total": 49714.73,
-  "detailedroute__mem__peak": 37619264.0,
-  "finish__clock__latency__min": 317.95,
-  "finish__clock__latency__max": 429.9,
-  "finish__clock__skew__worst": 97.0,
+  "detailedroute__runtime__total": "2:21:37",
+  "detailedroute__cpu__total": 55109.01,
+  "detailedroute__mem__peak": 39265476.0,
+  "finish__clock__latency__min": 328.46,
+  "finish__clock__latency__max": 449.6,
+  "finish__clock__skew__worst": 106.24,
   "finish__timing__setup__tns": 0.0,
-  "finish__timing__setup__ws": 519.93,
+  "finish__timing__setup__ws": 420.6,
   "finish__timing__drv__max_slew": 0.0,
-  "finish__timing__drv__max_fanout": 1831.0,
+  "finish__timing__drv__max_fanout": 1839.0,
```

Comment:
do you know the reason for the high number?

Reply:
No needs investigation.

Reply:
I agree it needs investigation but the change due to this PR is minimal.